docs/software/matlabinteractive.md

```diff
@@ -3,58 +3,225 @@ title: MATLAB Interactive
+
+```
+[clusteruser@login02 ~]$ egrep -o 'http(s)?://[a-zA-Z0-9.]+:[0-9]+' slurm-4089300.out 
+http://localhost:37851
+```
+
+Open an ssh tunnel to the appropriate compute node and port.
+
+
+```
+[localuser@localmachine ~]$ ssh -L 37851:node15:37851 login3.chpc.wustl.edu
+[clusteruser@login02 ~]$ # use Matlab
+[clusteruser@login02 ~]$ exit # close the tunnel when finished
+```
+
+Copy and paste the access URL into your local web browser. End the job when you are finished using Matlab.
+
+```
+[clusteruser@login01 ~]$ scancel 4089300
+```
+
+## Jupyter Notebook
+
+[Jupyter](https://jupyter.org/) is a web app for interactive notebooks containing both code and code output, such as matrices and plots. Jupyter was originally written for Python code, but there is an officially-supported Matlab kernel as well. Think of it as a `*.m` script you can step through one code block at a time, saving the outputs to review later and regenerate with just one click.
+
+To use the Jupyter Matlab kernel, follow our instructions for [Jupyter](jupyter-notebook.md). Install the `jupyter-matlab-proxy` package. For example, using a virtual environment and pip:
+
+```
+[clusteruser@node31 ~]$ source .venv/bin/activate
+(.venv) [clusteruser@node16 ~]$ pip install jupyter-matlab-proxy
+```
+
+Start the Jupyter Lab server and connect to it in your web browser as usual. You should see something like this.
+
+![Screenshot of Jupyter Lab launcher with Matlab available.](../images/jupyter-matlab-screenshot-1.png)
+
+Clicking `Open MATLAB` will open a new browser tab with the Matlab Proxy web application, the same one you can install standalone (without Jupyter) in the [previous section](#web-application). Clicking `MATLAB Kernel` will open a new Jupyter notebook (`*.ipynb` file) with a Matlab kernel. You can write code blocks in Matlab and run them just as you would a Python-based notebook.
+
+![Screenshot of Jupyter notebook with Matlab kernel.](../images/jupyter-matlab-screenshot-2.png)
+
+If you see a prompt like this, select the `Existing License` tab and then click `Start MATLAB`.
+
+![Configuring Matlab license](../images/jupyter-matlab-screenshot-3.png)
+
+## See Also
+
+- [Importing and Exporting Data](../getting-started/import-export-data.md) from outside the cluster. In particular, see the section on [using sshfs](../getting-started/import-export-data.md#sshfs-1), [from a worker node](../getting-started/import-export-data.md#controlling-transfers-using-a-worker-node), to mount your local computer as a network filesystem on the cluster.
+
+## X11 Forwarding
+
+We *do not* recommend using the Matlab graphical user interface with X11 forwarding due to performance, reliability, and stability issues. Instead, consider using the web application or Jupyter notebook interface above. 
+
+```
+[clusteruser@login01 ~]$ srun -p test -N 1 -n 8 --time=04:00:00 --pty --x11 bash
```

Comment:
update of command line params in future PR

Reply:
Updated in PR #23 via force-push